src/Redis/RedisClientAdapter.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Macpaw\RedisSchemaBundle\Redis;
+
+use Macpaw\SchemaContextBundle\Service\BaggageSchemaResolver;
+use Predis\Command\CommandInterface;
+use Predis\Command\FactoryInterface;
+use Predis\Configuration\OptionsInterface;
+use Predis\Connection\ConnectionInterface;
+use SymfonyBundles\RedisBundle\Redis\ClientInterface;
+
+class RedisClientAdapter implements ClientInterface
```

Comment:
```suggestion
use SymfonyBundles\RedisBundle\Redis\ClientInterface;

/**
 * @mixin ClientInterface
 */
class RedisClientAdapter implements ClientInterface
```
тут не треба таке додати? чи і без міксіна норм бачать магію?

Reply:
є опис методі в інтерфейсі Predis 
тому не потрібно, підхоплює ide нормально 
https://github.com/predis/predis/blob/main/src/ClientInterface.php#L56-L414